os/pm/pm_suspend.c

```diff
@@ -111,9 +111,17 @@ int pm_suspend(enum pm_domain_e domain)
 		set_errno(ERANGE);
 		goto errout;
 	}
+	/* prevent board going to sleep */
+	up_board_sleep(false);
```

Comment:
Can we contorl board sleep only using the pm state?

Reply:
Yes